modules/microvm/power-control.nix

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  ...
+}: let
+  cfg = config.ghaf.host.powercontrol;
+in
+  with lib; {
+    options.ghaf.host.powercontrol.enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = "Enable host power control";
+    };
+
+    config = mkIf cfg.enable {
+      ghaf.host.networking.enable = true;
```

Comment:
why setting this under power control it is already set in each target/definition

Reply:
Power control cannot work without network, so I enable it as a depedency. Maybe here some other ways to require network or indicate it needs the network?

Reply:
Done